diff --git a/kernel/pf_ring.c b/kernel/pf_ring.c
index cbaa8d61..f0f42c59 100644
--- a/kernel/pf_ring.c
+++ b/kernel/pf_ring.c
@@ -59,7 +59,7 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software Foundation,
- * Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ * Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA...
  *
  */
 
